TaskCalculator/src/CalculatorSwitch.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner;
+
+public class CalculatorSwitch {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Write first value");
+        double firstValue = sc.nextDouble();
+        System.out.println("Choose operation: +, -, *, /, % ");
+        String operator = sc.next();
+        System.out.println("Write second value");
+        double secondValue = sc.nextDouble();
+        double result;
+        switch (operator) {
+            case "+" -> {
+                result = firstValue + secondValue;
+                System.out.println("Result is: " + result);
+            }
+            case "-" -> {
+                result = firstValue - secondValue;
+                System.out.println("Result is: " + result);
+            }
+            case "/" -> {
+                if (secondValue == 0) {
+                    System.out.println("Division by '0' is impossible. Try again");
```

Comment:
Please rewrite it like this, if I enter 0 then division will happen anyway
![image](https://user-images.githubusercontent.com/109154988/182873306-20341839-4295-49dc-8d0d-82dec5c9bfa7.png)